package.json

```diff
@@ -34,12 +34,12 @@
     "mocha-junit-reporter": "^1.17.0",
     "pegin-address-verificator": "git+https://git@github.com/rsksmart/pegin-address-verifier#v0.2.1",
     "peglib": "git+https://github.com/rsksmart/rsk-peglib#v1.4.15",
+    "precompiled-fingerroot500": "npm:@rsksmart/rsk-precompiled-abis@^5.0.0-FINGERROOT",
+    "precompiled-hop400": "npm:@rsksmart/rsk-precompiled-abis@^5.0.0-HOP",
+    "precompiled-iris300": "npm:@rsksmart/rsk-precompiled-abis@^4.0.0-IRIS",
     "precompiled-orchid": "npm:@rsksmart/rsk-precompiled-abis@^1.0.0-ORCHID",
-    "precompiled-wasabi100": "npm:@rsksmart/rsk-precompiled-abis@^2.0.0-WASABI",
     "precompiled-papyrus200": "npm:@rsksmart/rsk-precompiled-abis@^3.0.0-PAPYRUS",
-    "precompiled-iris300": "npm:@rsksmart/rsk-precompiled-abis@^4.0.0-IRIS",
-    "precompiled-hop400": "npm:@rsksmart/rsk-precompiled-abis@^5.0.0-HOP",
-    "precompiled-fingerroot500": "npm:@rsksmart/rsk-precompiled-abis@^5.0.0-FINGERROOT",
+    "precompiled-wasabi100": "npm:@rsksmart/rsk-precompiled-abis@^2.0.0-WASABI",
```

Comment:
Not a big deal, but I would like to see these dependencies ascent or descent order, but ordered